### PR TITLE
updated index.html

### PR DESCRIPTION
**Pull request by Couch3ater**
also added bootstrap libraries

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>fileFU  |  HOME</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.css">
+    <link rel="stylesheet" href="css/style.css">
```
Comment (sadfunction) [inline public/index.html]: You should add a link to the global.js file here as well. Makes the inline code below body tag extraneous. 

   `<script src="./javascript/global.js"></script>`

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    <div class="container">
+      <div class="row">
+        <div class ="col-md-6"><p><h1 class="text-center">fileFU</h1></div>
+      </div>
+
+      <div class="row">
+        <div class ="col-md-6"><p class="text-center">fileFU is a simple file server brought to you by Dragonegger2.</div>
+      </div>
+
+        <form method="post" action="/api/upload", enctype="multipart/form-data" style="padding-top:10">
```
Comment (sadfunction) [inline public/index.html]: Going to need id="form" on this tag for the current code to work.

File: public/index.html (hunk 1 above)
Comment (Couch3ater) [inline public/index.html]: Added ref to script in <head>

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+    <!-- The script below runs on document ready -->
+    <script>
+      $(document).ready(function(){
+        populateTable();
+      });
+    </script>
+
+    <!-- The script below updates the file input once a file is selected -->
+    <script>
```
Comment (Couch3ater) [inline public/index.html]: I've moved this into the global.js

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            <tbody>
+            </tbody>
+          </table>
+        </div>
+      </div>
+      </div>
+
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    <script src="/javascript/global.js"></script>
```
Comment (Couch3ater) [inline public/index.html]: Without this ref, the table does not seem to populate; my assumption -- initial ref handles the alert on page load while this ref populates the table once the page has drawn (it's sitting right before the </body>)

**Pull request merged**